src/utils/escape.js

```diff
@@ -5,18 +5,15 @@ const CSS = (global.CSS: any)
 
 const env = process.env.NODE_ENV
 
+const escapeRegex = /([[\].#*$><+~=|^:(),"'`])/g
+
 export default (str) => {
   // We don't need to escape it in production, because we are not using user's
-  // input for selectors, we are generating a selector completely.
+  // input for selectors, we are generating a valid selector.
   if (env === 'production') return str
 
   if (!CSS || !CSS.escape) {
-    warning(
-      false,
-      '[JSS] CSS.escape polyfill in DEV mode is required in this browser, ' +
-      'check out https://github.com/mathiasbynens/CSS.escape'
-    )
-    return str
+    return str.replace(escapeRegex, '\\$1')
```

Comment:
I haven't tested the `'\\$1'` solution. I trust you on this.

Reply:
I have a trivial test, which shows that it works for "()", I don't test each character though.

Reply:
That test compares it with CSS.escape.